fahrschein-opentelemetry/src/main/java/org/zalando/fahrschein/opentelemetry/OpenTelemetryWrapper.java

```diff
@@ -0,0 +1,72 @@
+package org.zalando.fahrschein.opentelemetry;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.zalando.fahrschein.domain.Metadata;
+
+import io.opentelemetry.api.GlobalOpenTelemetry;
+import io.opentelemetry.api.OpenTelemetry;
+import io.opentelemetry.api.trace.SpanContext;
+import io.opentelemetry.api.trace.Tracer;
+import io.opentelemetry.context.Context;
+import io.opentelemetry.context.propagation.TextMapGetter;
+import io.opentelemetry.context.propagation.TextMapPropagator;
+import io.opentelemetry.context.propagation.TextMapSetter;
+
+/**
+ * OpenTelemetry support class for Fahrschein Nakadi client. This class provides
+ * helper methods for setting up OpenTelemetry.
+ */
+public class OpenTelemetryWrapper {
```

Comment:
the way it's used (public static methods) rather suggests a "Utils" or "Helper" class name.